ci-operator/config/redhat-developer/odo/redhat-developer-odo-main__odo-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make bin
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/build-root/Dockerfile
+images:
+- context_dir: openshift-ci/build-root/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: oc-bin-image
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 800m
+      memory: 6Gi
+tests:
+- as: odo-scenario-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+    test:
+    - ref: odo-tests
```

Comment:
How is the operator being installed on the test cluster? Is it from the script used to execute the tests?

Reply:
Not an operator, binary. Yes it is from the script used to execute the tests.